specification/paloaltonetworks/resource-manager/PaloAltoNetworks.Cloudngfw/stable/2023-09-01/PaloAltoNetworks.Cloudngfw.json

```diff
@@ -0,0 +1,7252 @@
+          "LocalRulestacks_listAdvancedSecurityObjects_MaximumSet_Gen": {
+            "$ref": "./examples/LocalRulestacks_listAdvancedSecurityObjects_MaximumSet_Gen.json"
+          },
+          "LocalRulestacks_listAdvancedSecurityObjects_MinimumSet_Gen": {
+            "$ref": "./examples/LocalRulestacks_listAdvancedSecurityObjects_MinimumSet_Gen.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/PaloAltoNetworks.Cloudngfw/localRulestacks/{localRulestackName}/listAppIds": {
+      "post": {
+        "tags": [
+          "LocalRulestacks"
+        ],
+        "operationId": "LocalRulestacks_listAppIds",
+        "description": "List of AppIds for LocalRulestack ApiVersion",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/LocalRulestackResource.localRulestackName"
+          },
+          {
+            "name": "appIdVersion",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "appPrefix",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "skip",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "top",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "format": "int32"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/ListAppIdResponse"
+            }
+          },
+          "default": {
+            "description": "Common error response for all Azure Resource Manager APIs to return error details for failed operations.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "LocalRulestacks_listAppIds_MaximumSet_Gen": {
+            "$ref": "./examples/LocalRulestacks_listAppIds_MaximumSet_Gen.json"
+          },
+          "LocalRulestacks_listAppIds_MinimumSet_Gen": {
+            "$ref": "./examples/LocalRulestacks_listAppIds_MinimumSet_Gen.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink",
+          "operationName": "LocalRulestacks_listAppIds"
```

Comment:
Could you remove this operationName?

Reply:
@Alancere These changes are part of approved preview version https://github.com/Azure/azure-rest-api-specs/pull/25772/files
What I can recall, this was introduced since by default powershell uses GET on nextLink but our API is POST.